conf/.gitignore

```diff
@@ -0,0 +1 @@
+/ds-image-kaltura-integration-test.yaml
```

Comment:
Could we maybe add the values in these added git ignores to the overall .gitignore. 
I dont think we need three ignore files here

Reply:
fixed